docker/pravega/scripts/init_kubernetes.sh

```diff
@@ -47,17 +47,20 @@ init_kubernetes() {
         local podname=${POD_NAME}
         export PUBLISHED_ADDRESS=""
         export PUBLISHED_PORT=""
-
+	
+	export PUBLISHED_ADDRESS=$( k8 "${ns}" "services" "${podname}" ".metadata.annotations[\"external-dns.alpha.kubernetes.io/hostname\"]" )
```

Comment:
The trailing period must be stripped (if any).

Reply:
Done.